core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4728,6 +4729,27 @@ public String toString() {
   //
   // FUSE integration related properties
   //
+  public static final PropertyKey FUSE_AUTH_POLICY_CLASS =
+      new Builder(Name.FUSE_AUTH_POLICY_CLASS)
+          .setDefaultValue("alluxio.fuse.auth.DefaultAuthPolicy")
+          .setDescription("The fuse auth policy class.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey FUSE_AUTH_POLICY_CUSTOM_USER =
+      new Builder(Name.FUSE_AUTH_POLICY_CUSTOM_USER)
+          .setDefaultValue("<FUSE_AUTH_POLICY_CUSTOM_USER>")
+          .setDescription("The fuse user name for custom auth policy.")
```

Comment:
Add a description showing this property will only be used when FUSE_AUTH_POLICY_CLASS is set to custom policy class.
same for the group property

Reply:
done